packages/template-retail-react-app/translations/en-US.json

```diff
@@ -89,6 +89,12 @@
   "account_order_detail.title.order_details": {
     "defaultMessage": "Order Details"
   },
+  "account_order_detail.toast.cancellation_failed": {
+    "defaultMessage": "Request cancellation did not succeed."
+  },
+  "account_order_detail.toast.cancellation_success": {
+    "defaultMessage": "Cancellation request submitted."
```

Comment:
How about the following:
"Your order cancellation request was submitted."

Reply:
I understand that order cancellation may not be immediately approved, so we can also say:

"Your order cancellation request is being processed."